clang/lib/Sema/SemaOpenACC.cpp

```diff
@@ -94,9 +94,10 @@ StmtResult SemaOpenACC::ActOnEndStmtDirective(OpenACCDirectiveKind K,
   case OpenACCDirectiveKind::Parallel:
   case OpenACCDirectiveKind::Serial:
   case OpenACCDirectiveKind::Kernels:
+    // TODO OpenACC: Add clauses to the construct here.
     return OpenACCComputeConstruct::Create(
         getASTContext(), K, StartLoc, EndLoc,
-        AssocStmt.isUsable() ? AssocStmt.get() : nullptr);
+        /*Clauses=*/{}, AssocStmt.isUsable() ? AssocStmt.get() : nullptr);
```

Comment:
```suggestion
        /*Clauses=*/std::nullopt, AssocStmt.isUsable() ? AssocStmt.get() : nullptr);
```

Reply:
Making the change (assuming it builds:) )  But curious as to why the nullopt preference here?  Both create an 'empty' `ArrayRef`, right?

Reply:
Just preferred way for constructing empty ArrayRef

Reply:
Ah, ok then :)  Patch incoming.